src/workerd/api/node/tests/net-nodejs-tcp-server.js

```diff
@@ -0,0 +1,17 @@
+const net = require('node:net');
+
+const server = net.createServer(function (s) {
+  s.end();
+});
+server.listen(9999, () => console.info('Listening on port 9999'));
+
+const echoServer = net.createServer(function (s) {
+  s.setEncoding('latin1');
+  s.on('data', function (chunk) {
+    s.write(chunk, 'latin1');
+  });
+  s.on('end', function () {
+    s.end();
+  });
+});
```

Comment:
If the intention is for this server to be started once and reused across multiple tests, I would recommend adding `error` event handlers to things so that things like abruptly disconnected streams don't cause the process to crash.